apps/browser/src/platform/system-notifications/browser-system-notification.service.ts

```
@@ -0,0 +1,90 @@
+    private platformUtilsService: PlatformUtilsService,
+  ) {
+    this.notificationClicked$ = merge(
+      fromChromeEvent(chrome.notifications.onButtonClicked).pipe(
+        map(([notificationId, buttonIndex]) => ({
+          id: notificationId,
+          buttonIdentifier: buttonIndex,
+        })),
+      ),
+      fromChromeEvent(chrome.notifications.onClicked).pipe(
+        map(([notificationId]: [string]) => ({
+          id: notificationId,
+          buttonIdentifier: ButtonLocation.NotificationButton,
+        })),
+      ),
+    );
+  }
+
+  async create(createInfo: SystemNotificationCreateInfo): Promise<string | undefined> {
+    try {
+      const notificationId = createInfo.id || uuidv4();
+
+      chrome.notifications.create(notificationId, {
+        iconUrl: "https://avatars.githubusercontent.com/u/15990069?s=200",
+        message: createInfo.body,
+        type: "basic",
+        title: createInfo.title,
+        buttons: createInfo.buttons.map((value) => {
+          return { title: value.title };
+        }),
+      });
+
+      // eslint-disable-next-line no-restricted-syntax
+      chrome.notifications.onButtonClicked.addListener(
+        (notificationId: string, buttonIndex: number) => {
+          this.notificationClicked$.subscribe({
+            next: () => ({
+              id: notificationId,
+              buttonIdentifier: buttonIndex,
+            }),
+          });
+        },
+      );
+
+      // eslint-disable-next-line no-restricted-syntax
+      chrome.notifications.onClicked.addListener((notificationId: string) => {
+        this.notificationClicked$.subscribe({
+          next: () => ({
+            id: notificationId,
+            buttonIdentifier: ButtonLocation.NotificationButton,
+          }),
+        });
+      });
+
+      return notificationId;
+    } catch (e) {
+      this.logService.error(
+        `Failed to create notification on ${this.platformUtilsService.getDevice()} with error: ${e}`,
+      );
```

I'd rather we just throw the error back to the caller than cover it up.

---

fixed with [49bef8e](https://github.com/bitwarden/clients/pull/15611/commits/49bef8ee3c5e53ea6b5b22d15e554648242a17ea)